### ISSUE TITLE
ag supports PCRE

### ISSUE THREAD

**Issue by datn** [beyondgrep/website#117]
To be fair to ag, it supports PCRE (in fact, it's a prereq):

https://github.com/ggreer/the_silver_searcher/wiki

**Comment by petdance** [comment]
It's right there at the top, fifth row down.  The row "Regular expressions" says "Perl-compatible".  Or is there something I'm missing?

**Comment by datn** [comment]
You're right -- it's a very small thing, but under the "Regular expressions" section (after the "Searching" section), the checkbox for "Use Perl (or Perl-compatible) regular expressions" is not checked for `ag`.

**Comment by petdance** [comment]
Aha. Thanks.
